### PR TITLE
Pag-1168 paGetPaymentRes - Test sintattici ko

### PR DESCRIPTION
**Pull request by davideromagnolieveris**
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+                      <value>22</value>
+                   </mapEntry>
+                </metadata>
+             </data>
+          </paf:paGetPaymentRes>
+       </soapenv:Body>
+    </soapenv:Envelope>
+    """
+    And <elem> with <tagvalue> in paGetPayment
+    And if outcome is KO set fault to None in paGetPayment
```
Comment (jacopocarlini) [inline src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature:98]: @davideromagnolieveris is this step correct?
`fault` tag is not present in the the paGetPayment XML

File: src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+      | description                 | Empty                                                                                                                                                                                                                                                             | SIN_PGPR_33   |
+      | description                 | test di prova per una lunghezza superiore a 140 caratteri alfanumerici, per verificare che il nodo risponda PPT_STAZIONE_INT_PA_ERRORE_RESPONSE                                                                                                                   | SIN_PGPR_34   |
+      | paymentCategory             | None                                                                                                                                                                                                                                                              | SIN_PGPR_35   |
```
Comment (jacopocarlini) [inline src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature:215]: `paymentCategory` is not present in the `paGetPayment` XML

File: src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+      | fullName                    | None                                                                                                                                                                                                                                                              | SIN_PGPR_56   |
+      | fullName                    | Empty                                                                                                                                                                                                                                                             | SIN_PGPR_57   |
+      | fullName                    | lunghezza >16char                                                                                                                                                                                                                                                 | SIN_PGPR_58   |
```
Comment (jacopocarlini) [inline src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature:234]: this test fails

File: src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature (hunk 2 above)
Comment (davideromagnolieveris) [reply src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature:215]: You're right, in the latest version of the paGetPayment XML this tag has been deleted

File: src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature (hunk 3 above)
Comment (davideromagnolieveris) [reply src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature:234]: Insert string "test di prova per una lunghezza superiore a 70 caratteri alfanumerici ," in line 234

File: src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature (hunk 1 above)
Comment (davideromagnolieveris) [reply src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature:98]: Wrong XML paGetPayment; insert the following:
`"""
    <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:paf="http://pagopa-api.pagopa.gov.it/pa/paForNode.xsd">
       <soapenv:Header/>
       <soapenv:Body>
          <paf:paGetPaymentRes>
             <outcome>#outcome#</outcome>
             <fault>
                <faultCode>#faultCode#</faultCode>
                <faultString>#faultString#</faultString>
                <id>#id#</id>
                <description>#description#</description>
             </fault>
          </paf:paGetPaymentRes>
       </soapenv:Body>
    </soapenv:Envelope>
    """`

**Review by jacopocarlini — APPROVED** [review]
all tests pass 👍

**Pull request merged**